python/ray/dashboard/http_server_head.py

```diff
@@ -139,6 +140,68 @@ async def get_favicon(self, req) -> aiohttp.web.FileResponse:
             )
         )
 
+    def get_gmt_offset(self):
+        current_tz = datetime.now().astimezone().tzinfo
+        offset = current_tz.utcoffset(None)
+        hours, remainder = divmod(offset.total_seconds(), 3600)
+        minutes = remainder // 60
+        sign = "+" if hours >= 0 else "-"
+        return f"GMT{sign}{abs(int(hours)):02d}:{abs(int(minutes)):02d}"
```

Comment:
let's move this and most of the implementation of `get_timezone` into a separate file. SOmething like `timezone_utils.py`